openassessment/workflow/models.py

```diff
@@ -179,6 +179,21 @@ def score(self):
         """
         return sub_api.get_latest_score_for_submission(self.submission_uuid)
 
+    @property
+    def override_score(self):
```

Comment:
Not that keen on renaming since downstream it is the override score and I don't really want to change it's name as it goes through the pipeline.